ui/admin/tests/unit/controllers/scopes/scope/targets/index-test.js

```diff
@@ -0,0 +1,12 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'admin/tests/helpers';
+
+module('Unit | Controller | scopes/scope/targets/index', function (hooks) {
+  setupTest(hooks);
+
+  // TODO: Replace this with your real tests.
```

Comment:
Can we try adding some tests for our getters and actions in the controller? This will help verify they are all performing in the way we expect.

Reply:
very good point, updated!